cylc/flow/__init__.py

```diff
@@ -58,11 +58,9 @@ def environ_init():
 
 def iter_entry_points(entry_point_name):
     """Iterate over Cylc entry points."""
-    import pkg_resources
+    from importlib_metadata import entry_points
```

Comment:
Shouldn't this use the same `importlib.metadata` vs `importlib_metadata` logic as in `cylc.py`?